### PR TITLE
Consume Leaderboard API

### PR DESCRIPTION
**Pull request by tkz96**
This PR is to consume the Leaderboard API

### CODE REVIEW

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## Hi @tkz96  👋 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that should have been submitted for consuming API project but you submitted it for final touches that is the last milestone for the leaderboard project. Please resubmit it for consume API project. Good luck 👍

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/games.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import renderTable from './renderTable.js';
+
+const FINAL_URL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/iLhUHUKbK0GFKQnhs8n8/scores/';
+
+const apiRefreshScores = async () => {
+  await fetch(`${FINAL_URL}`)
+    .then((response) => response.json())
+    .then((json) => renderTable(json));
+};
```
Comment (VICTOR-CHUKWU) [inline src/games.js]: - [ ] According to the project requirement, use async and await your promise, your code can be cleaner without `then` method. you can change it to something like this :point_down: 
```
const apiRefreshScores = async () => {
 let data = await fetch(`${FINAL_URL}`)
 let resp = await data.json()
 renderTable(resp)
};
```

File: src/games.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    headers: {
+      'Content-type': 'application/json',
+    },
+    body: JSON.stringify({
+      user: _name,
+      score: _score,
+    }),
+  }).then((response) => response.json());
+  apiRefreshScores();
+};
```
Comment (VICTOR-CHUKWU) [inline src/games.js]: - [ ] Also use `await` method here to await your API response

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @tkz96  👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good PR
- Good commit
- All linters passed


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @tkz96, 

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**